tests/tests.js

```diff
@@ -1,45 +1,204 @@
+    })
   })
 
-  it('throws an error on unknown product type', () => {
-    const unknownProduct = { type: 'vision' }
+  describe('formatPrice', () => {
+    it('returns the price with two decimals not rounded', () => {
+      const formattedPrice = pricing.formatPrice(22.112284)
+
+      expect(formattedPrice).to.equal(22.11)
+    })
+
+  })
+
+  describe('calculateProductPrice', () => {
+    it('returns the price for a voluntary life product for a single employee', () => {
+
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      }
+      const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+
+      expect(price).to.equal(39.37)
+
+      expect(formatPriceSpy).to.have.callCount(1)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+      expect(calculateVolLifePriceSpy).to.have.callCount(1)
+    })
+
+    it('returns the price for a VolLife product for an employee with a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+
+      expect(price).to.equal(71.09)
+
+      expect(formatPriceSpy).to.have.callCount(1)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+      expect(calculateVolLifePriceSpy).to.have.callCount(1)
+    })
+
+    it('returns the price for a disability product for an employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee']
+      }
+      const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(22.04)
+
+      expect(formatPriceSpy).to.have.callCount(1)
+      expect(getEmployerContributionSpy).to.have.callCount(1)
+      expect(calculateLTDPriceSpy).to.have.callCount(1)
+    })
+
+    it('throws an error on unknown product type', () => {
+      const unknownProduct = { type: 'vision' }
+
+      expect(() => pricing.calculateProductPrice(unknownProduct, {}, {})).to.throw('Unknown product type: vision')
+    })
+  })
+
+  describe('calculateCommuterPrice', () => {
+    it('returns the price for a train commuter', () => {
+      const selectedOptions = {
+        benefit: 'train'
+      }
+
+      const price = pricing.calculateCommuterPrice(products.commuter, selectedOptions)
+
+      expect(price).to.equal(84.75)
+    })
+    it('returns the price for a parking commuter', () => {
+      const selectedOptions = {
+        benefit: 'parking'
+      }
+
+      const price = pricing.calculateCommuterPrice(products.commuter, selectedOptions)
 
-    expect(() => pricing.calculateProductPrice(unknownProduct, {}, {})).to.throw('Unknown product type: vision')
+      expect(price).to.equal(250)
+    })
```

Comment:
Nice unit tests. Don't forget the integration test for `calculateProductPrice` that will help make sure you wire things up correctly.